recipes-bsp/uefi/edk2-hikey_git.bb

```diff
@@ -21,7 +21,7 @@ SRCREV_atffastboot = "5b0d44c057bc0005965da990e8add72670810996"
 # knowledgeably, it is a hack...
 ###
 SRC_URI = "git://github.com/96boards-hikey/edk2.git;name=edk2;branch=testing/hikey960_v2.5 \
-           git://github.com/ARM-software/arm-trusted-firmware.git;name=atf;branch=integration;destsuffix=git/atf \
+           git://github.com/ARM-software/arm-trusted-firmware.git;name=atf;branch=master;destsuffix=git/atf \
```

Comment:
The branch name is changed. Is there any reason that we choose master branch?

Reply:
yes, master is preferred than integration. As of today, master was matching integration so I took th opportunity to switch.